Source/SwiftLintFramework/Rules/VariableNameRule.swift

```diff
@@ -16,13 +16,15 @@ public struct VariableNameRule: ASTRule {
         identifier: "variable_name",
         name: "Variable Name",
         description: "Variable name should only contain alphanumeric characters and " +
-          "start with a a lowercase character. In an exception to the above, variable " +
-          "names may start with a capital letter when they are declared static and immutable.",
+          "start with a a lowercase character or should only contain capital letters. " +
```

Comment:
"a a". I know it was already like that but as you're already modifying this line, I'd appreciate if you could fix this typo. Thanks!